docs/CloudPermissions.md

```diff
@@ -0,0 +1,78 @@
+# Assign Required Cloud Permissions For Deploying Scalar DL
+
+This guide explains how to assign required permissions for the users to create cloud resources for deploying Scalar DL on Azure.
+
+From the security perspective, it is better to give users limited permissions to protect the environment from unwanted activities.
+
+## Azure
+
+The following JSON is a custom role that allows users to manage Azure cloud resources from the Azure portal or from command line tools for deploying Scalar DL.
+Please note that it is sufficient but not necessary since, it can/should be further restricted.
```

Comment:
Then I think it should be more restricted, as you say...
If you let users do, then it defeats the purpose of the doc.

Reply:
For this, it can be done later on. For now, it's fine since we don't have time.

Reply:
OK.